examples/compiler/02_data_regions.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this tutorial we will learn about data regions and how these impact the Operator construction. We will use the simple time marching example shown in the [01_iet](https://github.com/opesci/devito/blob/master/examples/compiler/01_iet.ipynb) tutorial. "
+    "In this tutorial we will learn about data regions and how these impact the Operator construction. We will use the simple time marching example."
```

Comment:
why this change?

Reply:
because this filename will change.
not only that, but probably this notebook will come first than the notebook on the IET (see the index in [00_index.ipynb](https://github.com/opesci/devito/blob/tutorial_usp-2/examples/compiler/00_index.ipynb)

Reply:
then we need to move it in here, don't we?

also, I think we can infer what the filename will be -- it's pretty much the same string that is in current master, except for the actual filename that changes. So I'd use it and as soon as the branch lands, it should work; then we go check, and if we broke something, we can immediately file and merge a hotfix PR to tweaks the broken link (but again, this might not even be an issue)

Reply:
ok